Cargo.lock

```diff
@@ -4934,7 +4934,7 @@ dependencies = [
 
 [[package]]
 name = "zinnia"
-version = "0.21.1"
+version = "0.22.0"
```

Comment:
I think you should revert all of these version changes for local packages in this monorepo. They will conflict with the changes made on the `main` branch.

Reply:
Resolved in https://github.com/CheckerNetwork/zinnia/pull/691/commits/4d4d00e6c3b605b523d1d902b03a5700a3519f4e